repology/parser/slackware.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2016 Dmitry Marakasov <amdmi3@amdmi3.ru>
+# Copyright (C) 2017 David Spencer <idlemoor@slackbuilds.org>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import sys
+
+from repology.package import Package
+
+
+class SlackwareParser():
+    def __init__(self):
+        pass
+
+    def Parse(self, path):
+        packages = {}
+
+        # For all the Slackware packages:
+        maintainers = ['volkerdi@slackware.com']
+
+        # Packages are found in:
+        #   slackware*/  (but NOT kdei)
+        #   patches/
+        #   extra/       (but NOT aspell-word-lists)
+        # Everything else is ignored:
+        #   isolinux/, kernels/, pasture/, source/, testing/,
+        #   usb-and-pxe-installers/
+
+        with open(path, 'r') as filelisttxt:
+            for line in filelisttxt.readlines():
+                fields = line.strip().split('/')
+                if re.search('\.t.z$', fields[-1]):
+                    revsplit = fields[-1][::-1].split('-', 3)
+                    if len(revsplit) != 4:
+                        continue
+                    pkgnam = revsplit[3][::-1]
+                    version = revsplit[2][::-1]
+
+                    if fields[1][:9] == 'slackware' and fields[2] != 'kdei':
```

Comment:
I don't see `slackware*` there, only `slackware`, so this could be simplified. Why exccude kdei? These may be merged into a single package kde-l10n or kde-i18n like we do with other repos.